axiom/monitors_integration_test.go

```diff
@@ -118,3 +118,24 @@ func (s *MonitorsTestSuite) Test() {
 
 	s.Contains(monitors, s.monitor)
 }
+
+func (s *MonitorsTestSuite) CreateMatchMonitor() {
+	monitor, err := s.client.Monitors.Create(s.ctx, axiom.MonitorCreateRequest{
+		Monitor: axiom.Monitor{
+			AlertOnNoData: false,
+			APLQuery:      fmt.Sprintf("['%s']", s.datasetID),
+			Description:   "A very good test monitor",
+			DisabledUntil: time.Now().Add(time.Minute * 10),
+			Interval:      time.Minute,
+			Name:          "Test Monitor",
+			Operator:      axiom.BelowOrEqual,
+			Range:         time.Minute * 10,
+			Threshold:     5,
+			Type:          axiom.MatchEvent,
+		},
+	})
+	s.Require().NoError(err)
+	s.Require().NotNil(monitor)
+
+	s.Equal(monitor.Type.String(), "MatchEvent")
+}
```

Comment:
Does `testify` call methods no prefixed with `Test...`? I don't think so because this is missing teardown code and every invocation of this test should create a new monitor but not delete it. And I see no monitors in our integration testing env.